kuma/wiki/migrations/0004_revision_akismet_submission_on_delete_cascade.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration should be manually merged with 0003_remove_soft_document_deletion.py